kuzzle-sdk-java/src/main/java/io/kuzzle/sdk/Events/EventListener.java

```diff
@@ -0,0 +1,74 @@
+package io.kuzzle.sdk.Events;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Consumer;
+
+public class EventListener<T> {
```

Comment:
Because I copied the behavior of C# events, I can't remember another reason

Reply:
But the main advantage is that you can't misspell the event name :smile:

Reply:
We have this behavior in C# because the language natively support events.  
With the current implementation, we will have 2 methods per event which is, IMHO, not very convenient to maintain and to document.
It will be easier to have some kind of `kuzzle.addListener` method.
I think we need a workshop about this.